dist/626.js.map

```diff
@@ -0,0 +1 @@
          let thread_grouped_z2 = thread_group_func(MathHelper.zip(l3misses, tot_cyc));\n            let l3corr = thread_grouped_z2.map(x => MathHelper.sample_corr(x[0], x[1]));\n\n            // Now we average the correlations\n            let avg_l2_corr = MathHelper.mean(l2corr);\n            let avg_l3_corr = MathHelper.mean(l3corr);\n\n            let i = 0;\n            for (let tcm of [l2misses, l3misses]) {\n                let labelstr = \"[Unknown]\";\n                if(tcm == l2misses) labelstr = \"L2_TCM\";\n                if(tcm == l3misses) labelstr = \"L3_TCM\";\n\n                let tmp = 0;\n\n                if(toplevel_use_mean)\n                    tmp = MathHelper.zip2d(tcm).map(x => MathHelper.mean(x));\n                else if(toplevel_use_median)\n                    tmp = MathHelper.zip2d(tcm).map(x => MathHelper.median(x));\n                else ObjectHelper.assert(\"Undefined mode\", false);\n\n                datasets.push({ label: labelstr, xAxisID: \"data-axis\", yAxisID: \"thread-axis\", data: tmp, backgroundColor: colors[i] });\n                i++;\n            }\n\n            let rho_col1 = colors[i];\n            let rho_col2 = colors[i + 1];\n\n            if(display_memory_correlation) {\n                datasets.push({ label: \"L2 corr.\", xAxisID: \"corr-axis\",/* yAxisID: \"corr-y-axis\",*/ data: l2corr, backgroundColor: colors[i], hidden: true });\n                datasets.push({ label: \"L3 corr.\", xAxisID: \"corr-axis\", /*yAxisID: \"corr-y-axis\",*/ data: l3corr, backgroundColor: colors[i + 1], hidden: true });\n            }\n\n            let chartData = {\n                labels: [...(Array(l2misses[0].length).keys())].map(x => \"Thread \" + x.toString()),\n                \"datasets\": datasets,\n\n            };\n\n            return chartData;\n        }).linkMouse(this.button_subwindow).changeGraphOptions(x => {\n            x.options.title.text = \"Cache misses (mean over all runs)\";\n            x.options.title.display = false;\n            x.options.legend = {\n                position: 'top',\n                display: true\n            };\n        }));\n\n        \n        databinding[\"PercentBandwidth\"] = all_mem_analyses;\n        databinding[\"Title\"] = new DataBlock({ fontsize: 32, text: \"Memory performance\", color: \"black\", align: \"center\" }, \"Text\");\n        databinding['Graph'] = all_mem_analyses;\n        databinding['Bandwidth'] = all_mem_analyses;\n\n        layout.setDataBinding(databinding);\n\n\n        this.button_subwindow.setLayout(layout);\n\n        this.setOnEnterHover(p => { this.color = \"#FF0000\"; this.button_subwindow_state = 'open'; })\n        this.setOnLeaveHover(p => { this.color = \"orange\"; if (!this.is_locked_open) this.button_subwindow_state = 'collapsed'; })\n        this.setOnClick((p, mb) => { this.is_locked_open = !this.is_locked_open; });\n\n        this.setDefaultDblClick();\n    }\n\n\n\n}\n\nexport { MemoryButton };"],"sourceRoot":""}
```

Comment:
They're for lazy-loading scripts. JavaScript does that by calling `import(...)` at runtime, and there's some DIODE code which makes use of that. Since Webpack bundles scripts together, if `import(...)` is found it creates lazy-loadable bundles containing all the lazily-loaded dependencies.

Removing DIODE from the codebase gets rid of them.